adept/_vlasov1d/modules.py

```diff
@@ -188,19 +188,16 @@ def init_state_and_args(self) -> Dict:
         for field in ["a", "da", "prev_a"]:
             state[field] = jnp.zeros(self.cfg["grid"]["nx"] + 2)  # need boundary cells
 
+        if self.cfg["terms"]["diags"]:
+            state["diag-vlasov"] = jnp.zeros_like(f)
```

Comment:
I think a more descriptive name might be `diag-vlasov-dfdt` and `diag-fp-dfdt`

Reply:
agreed...